src/index.ts

```diff
@@ -68,14 +69,19 @@ const PluginsHandler = {
 const Plugins = new Proxy(_ADJS.Plugins, PluginsHandler);
 const Networks = new Proxy(_ADJS.Networks, PluginsHandler);
 const Vendors = new Proxy(_ADJS.Vendors, PluginsHandler);
+const Debug = async () => {
```

Comment:
We aren’t writing any code that waits on debug. Because of this lets make it a standard function and not await load script. This will ensure that if we transpile to lower browsers that it doesnt become a regenerator function

Reply:
Totally. Done